src/jit/gentree.h

```diff
@@ -4782,6 +4782,23 @@ struct GenTreeJumpCC final : public GenTree
 #endif // DEBUGGABLE_GENTREE
 };
 
+struct GenTreeSelectCC final : public GenTreeOp
```

Comment:
I wonder if we could/should try to do this without adding classes like `GenTreeSelectCC` and `GenTreeJumpCC`. There are 6 conditions, they could be stored in 3 bits somewhere in `GenTree`. Probably it's not worth the trouble...